dowhy/causal_model.py

```diff
@@ -191,6 +192,20 @@ def identify_effect(
         if estimand_type is None:
             estimand_type = self._estimand_type
 
+        all_graph_variables = set.union(
```

Comment:
I think you could check all variables by calling causalgraph.py CausalGraph::get_all_nodes(...)

Reply:
That's easier indeed, thanks for pointing this out. 
Resolved now.